ext/opentelemetry-ext-django/setup.cfg

```diff
@@ -0,0 +1,55 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[metadata]
+name = opentelemetry-ext-django
+description = OpenTelemetry Instrumentor for Django
```

Comment:
```suggestion
description = OpenTelemetry Django integration
```

Reply:
I have changed this to `Instrumentation`. I rather use one term only.

Reply:
Has this term been finalized as the recommended language we are using? We are not using `integration` anymore?